armory/docker/management.py

```diff
@@ -2,15 +2,12 @@
 Docker orchestration managers for ARMORY.
 """
 
-import logging
 
 import docker
 
 import armory
 from armory import paths
-
-
-logger = logging.getLogger(__name__)
+from armory.logs import log as logger
```

Comment:
This should follow the same syntax, using `log` instead of `logger`:
```
from armory.logs import log
```

The use of `log` as a local variable in `exec_cmd` can easily be changed to `exec_stream` or something like that.

Reply:
Yep, there were two ways to clear this collision, but your consistency claim wins.